services/system/Transact/src/RTransact.cpp

```diff
@@ -781,15 +792,18 @@ void RTransact::onBlock()
       }
    }
 
-   PSIBASE_SUBJECTIVE_TX
+   if (isBooted())
```

Comment:
It might be better to just exclude the bootTransactions. Other transactions can expire normally.